src/pywrangler/cli.py

```diff
@@ -108,8 +107,7 @@ def sync_command(force=False):
 
     Also creates a virtual env for Workers that you can use for testing.
     """
-    # Check if pyproject.toml exists and requirements.txt does not
-    check_pyproject_toml()
+    # Check if requirements.txt does not exist.
     check_requirements_txt()
```

Comment:
Surely we should be checking for a `requirements.txt` next to the `pyproject.toml`?

Reply:
Yeah, good catch

Reply:
Actually, there is no bug here. `check_requirements_txt` uses `PROJECT_ROOT`.